sources/HUBViewControllerImplementation.m

```diff
@@ -864,6 +870,7 @@ - (void)createCollectionViewIfNeeded
     collectionView.dataSource = self;
     collectionView.delegate = self;
     
+    [self.view addGestureRecognizer:collectionView.panGestureRecognizer];
```

Comment:
Nope, since a gesture recognizer can only be part of a single view, it will be removed from the collection view 🙂